packages/components/src/entities/SamplesTabbedGridPanel.tsx

```diff
@@ -342,8 +343,10 @@ export const SamplesTabbedGridPanel: FC<Props> = memo(props => {
                     ButtonsComponent={gridButtons}
                     buttonsComponentProps={_gridButtonProps}
                     ButtonsComponentRight={SampleTabbedGridButtonsRight}
-                    supportedExportTypes={showLabelOption && canPrintLabels ? EXPORT_TYPES_WITH_LABEL : undefined}
-                    onExport={showLabelOption && canPrintLabels ? onLabelExport : undefined}
+                    // supportedExportTypes={showLabelOption && canPrintLabels ? EXPORT_TYPES_WITH_LABEL : undefined}
+                    // onExport={showLabelOption && canPrintLabels ? onLabelExport : undefined}
+                    supportedExportTypes={showLabelOption && canPrintLabels && !isAllTab ? EXPORT_TYPES_WITH_LABEL : undefined}
+                    onExport={showLabelOption && canPrintLabels && !isAllTab ? onLabelExport : undefined}
```

Comment:
I might capture the `showLabelOption && canPrintLabels && !isAllTab` as a variable, for better readability.

Reply:
done